css/types/image.json

```diff
@@ -405,6 +405,533 @@
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "44",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "layout.css.prefixes.webkit",
+                        "value_to_set": "true"
+                      }
+                    ]
+                  }
+                ],
+                "ie": {
+                  "version_added": "10",
+                  "notes": "Internet Explorer 5.5 through 9.0 supported gradients via a proprietary filter: <code><a href='https://developer.mozilla.org/docs/Web/CSS/-ms-filter#Gradient'>-ms-filter: progid:DXImageTransform.Microsoft.Gradient()</a></code>."
+                },
+                "opera": [
+                  {
+                    "version_added": "12.1"
+                  },
+                  {
+                    "prefix": "-o",
+                    "version_added": "11.6",
+                    "version_removed": "15"
+                  },
+                  {
+                    "prefix": "-webkit",
+                    "version_added": "15"
+                  }
+                ],
+                "opera_android": [
+                  {
+                    "version_added": "12.1"
+                  },
+                  {
+                    "prefix": "-o",
+                    "version_added": "12",
+                    "version_removed": "15"
+                  },
+                  {
+                    "prefix": "-webkit",
+                    "version_added": "15"
+                  }
+                ],
+                "safari": [
+                  {
+                    "version_added": "6.1"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "5.1",
+                    "notes": [
+                      "Safari 4 was supporting an experimental <code><a href='http://developer.apple.com/safari/library/documentation/InternetWeb/Conceptual/SafariVisualEffectsProgGuide/Gradients/Gradient.html'>-webkit-gradient(radial,…)</a></code> function. This old outdated syntax is still supported for compatibility purposes."
+                    ]
+                  }
+                ],
+                "safari_ios": {
+                  "version_added": true
+                },
+                "samsunginternet_android": {
+                  "version_added": true
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            },
+            "at": {
+              "__compat": {
+                "description": "<code>at</code> syntax",
+                "support": {
+                  "webview_android": {
+                    "version_added": true
+                  },
+                  "chrome": {
+                    "version_added": "26"
```

Comment:
I'm not as familiar with CSS content structure as I with APIs. If `at` gets its own page, then the webkit data needs to be copied.